ec2/spark_ec2.py

```diff
@@ -607,14 +608,17 @@ def get_existing_cluster(conn, opts, cluster_name, die_on_error=True):
             elif (cluster_name + "-slaves") in group_names:
                 slave_nodes.append(inst)
     if any((master_nodes, slave_nodes)):
-        print "Found %d master(s), %d slaves" % (len(master_nodes), len(slave_nodes))
+        print "Found %d master(s), %d slaves" % (len(master_nodes), len(slave_nodes)) \
+            + " in region " + opts.region
```

Comment:
Really minor point, but I would leave this print statement unchanged since users are unlikely to take action at this particular point, and we printed the region earlier.